en/reference/services-search.html

```diff
@@ -328,7 +329,63 @@ <h2 id="renderer">renderer</h2>
           bundle="the name in &lt;artifactId&gt; in pom.xml" /&gt;
 </pre>
 
+<h2 id="significance">significance</h2>
+<p>
+Contained in <a href="#searcher">searcher</a>.
+Specifies one or more global significance <a href="#model">model</a>.
+</p>
+
+<pre data-test="file" data-path="my-app/src/main/application/services.xml">
+&lt;significance&gt;
+    &lt;model model-id="wikimedia"/&gt;
+    &lt;model url="https://some/uri/my-model.model.multilingual.json"/&gt;
+    &lt;model path="models/my-model.no.json.zst"/&gt;
+&lt;/significance&gt;
+</pre>
+
+<p>
+The models are either provided by <em>Vespa</em> or generated with <a href="vespa-cmdline-tools.html#vespa-significance">vespa-signficance</a> CLI.
+The order determines model precedence - with the last element having the highest priority.
+To use these models, schema needs to <a href="schema-reference.html#significance">enable significance models in the rank-profile</a>.
+</p>
+
+<p>
+Sub-elements:
+<ul>
+    <li><a href="#model">model</a> (required, one or more)</li>
+</ul>
+</p>
+
+<h2 id="model">model</h2>
+<p>
+Contained in <a href="#siginficance">significance</a>.
+Specifies <a href="../significance.html#global-significance-model">global significance model</a>.
+Models are identified by <code>model-id</code> or by providing <code>url</code> or <code>path</code> to a model file in the application package.
+</p>
+<p>
+Models with <code>model-id</code> are provided by <em>Vespa</em>.
+So far the only model available is <em>wikimedia</em>, which is generated from English Wikipedia.
```

Comment:
I am not sure what will be the `id` of this model. I suggest we remove this section until those details are decided.

Reply:
Do you want to remove mentioning of model-id attribute or just the wikimedia example?

Reply:
We can remove the wikimedia example for now.